kitsune/questions/templates/questions/question_details.html

```diff
@@ -118,7 +118,9 @@ <h2 class="summary" itemprop="headline">{{ question.title }}</h2>
         </div>
         <div class="asked-on">
           {{ _('Posted') }}<br>
-          <time itemprop="dateCreated" datetime="{{ question.created }}">{{ datetimeformat(question.created) }}</time>
+          <span itemprop="dateCreated">
+            {{ datetimeformat(question.created) }}
+          </span>
```

Comment:
no we specifically want this in a `<time>` element. it is also a valid html5 tag.